docs/ui/auth/fragments/react/password-manager.md

```diff
@@ -0,0 +1,10 @@
+```jsx
+import {
+  AmplifyAuthContainer,
+  AmplifyAuthenticator,
+} from "@aws-amplify/ui-react";
+
+<AmplifyAuthContainer>
+  <AmplifyAuthenticator />
+</AmplifyAuthContainer>;
```

Comment:
Is the `;` here correct?

Reply:
Ah, I think prettier was confused here (which I'm not sure why it ran because fragements are under `.prettierignore`). Thanks for the catch! Will fix.